CMakeLists.txt

```diff
@@ -89,6 +89,7 @@ option(
   VELOX_ENABLE_INT64_BUILD_PARTITION_BOUND
   "make buildPartitionBounds_ a vector int64 instead of int32 to avoid integer overflow when the hashtable has billions of records"
   OFF)
+option(VELOX_ENABLE_QPL "Enable Intel QPL support" OFF)
```

Comment:
what is QPL? Can we rename it to something more intuitive, like `VELOX_ENABLE_INTEL_IAA` or similar?

Reply:
The Intel® Query Processing Library (Intel® QPL) is an open-source library to provide high-performance query processing operations on Intel CPUs.
QPL link: https://github.com/intel/qpl

I changed the '''VELOX_ENABLE_QPL''' to '''VELOX_ENABLE_INTEL_IAA '''